priv/repo/seeds.exs

```diff
@@ -14,7 +14,10 @@
 |> Trento.Users.User.changeset(%{
   username: "admin",
   password: "adminpassword",
-  confirm_password: "adminpassword"
+  confirm_password: "adminpassword",
+  fullname: "Trento Admin",
+  email: "admin@trento.com",
```

Comment:
Here we are using a different email than the release task hehe
In the release we have `admin@trento.suse.com`

Reply:
done